package.json

```diff
@@ -36,9 +36,11 @@
     "onCommand:quarkusTools.debugQuarkusProject",
     "onCommand:quarkusTools.welcome",
     "workspaceContains:**/src/main/resources/application.properties",
+    "workspaceContains:**/src/main/resources/application.yaml",
     "onLanguage:quarkus-properties",
     "onLanguage:java",
-    "onLanguage:qute"
+    "onLanguage:qute",
+    "onLanguage:yaml"
```

Comment:
that's probably too eager. See if you can remove it

Reply:
Removed, it seems it works without this yaml delclaration.